lib/browserify-plugins/id-hasher/id-hash-transform.js

```diff
@@ -2,12 +2,18 @@
 
 const common = require('asset-pipe-common');
 const { Transform } = require('readable-stream');
+const { relative } = require('path');
 
 module.exports = function idHashTransform(idToHashMap) {
     return new Transform({
         objectMode: true,
         transform(obj, enc, next) {
-            idToHashMap[obj.id] = common.hasher(obj.source);
+            // include id and source in hash to avoid situation where
+            // multiple modules have the same source code but different
+            // depedendencies somewhere down the line.
+            idToHashMap[obj.id] = common.hasher(
+                relative(process.cwd(), obj.id) + obj.source
```

Comment:
`obj.id` is the path of the module?

Reply:
yeah, full path on the machine that built it